### ISSUE TITLE
English translation

### ISSUE THREAD

**Issue by GianpaoloTedesco** [loiste-interactive/Obenseuer-Issues#2930]
Hello there, i'm a smalltime game dev from italy and i'm currently in love with your game. The only thing preventing me from enjoyng it to the fullest is the spelling and syntx errors in the dialogues/quasts/item descriptions. Since nobody was able to tell me if the errors are on puropse or not, i started compiling a list of items that need correpting, and the relative corrections. the document is public, anybody can access it. Let me know if the work i'm doing is appreciated or redundant. in case it isn't unnecessary, i'd start to work on the dialogues and quests aswell. lemmiekno.

doc link-->https://docs.google.com/spreadsheets/d/1BtRn-7QDC_BV5497EdHhBW9fELj2DsNUVGeMFpjV3ZE/edit?gid=0#gid=0

**Comment by Nextej** [comment]
I do feel like a lot of these kill the humours and silly tone the descriptions had brought with them, almost as if treating the player as an complete idiot, who needs explicit direct instructions on what an item in question is.

**Comment by efadfa3** [comment]
I also feel like some of them are just... not correct in the "corrected" version either. "a a4 piece of paper" is incorrect, the original is correct though. I do not understand your hatred for "it's." Also, it's macaroni. It's maccheroni in your native language, it's macaroni in English.

To be honest, for someone who is complaining about "syntx" (it's syntax) errors like a big grammar specialist, you're making a whole lot of typos and mistakes yourself. 

With all that being said, you are correct that the descriptions have a lot to be desired sometimes in the department of typos or grammar.

**Comment by GianpaoloTedesco** [comment]
I'm sorry but waht's with the hostility? I never said i was the absolute best, nor did i claim to be, criticism is always welcome and corrections too, honestly I left comments open on the doc just for that reason. Also, I'm just one person and I started writing today, i am not good enought to be able to produce perfect stuff on the first try.
About the Italian thing, you might be right, I just tought it was fitting, as the game is in Europe and already has a few Italian names in it but, for the sake of a translation in proper English, as I said, you're probbly right.
About typos in the post, I didn't spellcheck it, my bad I guess. Should be correct now.
About the hatred for the it's, I didn't remove it entirely, but I might have overcompensated a bit, i just noticed the fact that about half of the items had it's in the description and the other half didn't. It just irked me so i "corrected" it, but as you said i might have overdone it a fair bit.
Thanks for the spellcheck, i sincerely am grateful for that ,and the rest, honestly the more eyes look at it the better the doc will become. or so I believe.
Peace (also lemmiekno if you want access to the doc to modify it)

**Comment by efadfa3** [comment]
You're right, I came off too strong, I believed you were coming from a place of bad intentions, and it turned out that _**I**_ was. I would love the access to the doc. I'm sorry for being a bitch.

**Comment by GianpaoloTedesco** [comment]
Forgiven! I've given you access, idk if you recived a notification or not. Looking forward to...this? I don't exactly know what I wanted to say but i said it anyway. Happy writing, i guess? Idk
Peace!
